src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1546,14 +1548,14 @@ string GetParameterDescriptionForErrorMessage (ParameterDefinition parameterDefi
 			if (string.IsNullOrEmpty (parameterDefinition.Name))
 				return $"parameter #{parameterDefinition.Index} of method '{parameterDefinition.Method}'";
 
-			return $"parameter '{parameterDefinition.Name}' of method '{parameterDefinition.Method}'";
+			return $"parameter '{parameterDefinition.Name}' of method '{(parameterDefinition.Method as MethodDefinition).GetDisplayName ()}'";
```

Comment:
This is potentially dangerous. There doesn't seem to be a guarantee that the cast will succeed. So it should be conditional.